Update CONTRIBUTORS_QUICK_START.rst

OyinOlamide opened the pull request:
I am Oyinkansola Awosan, an Outreachy applicant for the December to March cohort. I noticed there was an error in the second block of code under the "Pyenv and setting up virtual-env" part of the guide. There was a letter 's' that was causing problems in the 'checking required packages' command.
I have corrected the error. This way other people who want to contribute to Apache won't face this issue I faced.
Thank you. :)

<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

OyinOlamide committed:
Update CONTRIBUTORS_QUICK_START.rst

I Oyinkansola Awosan, an Outreachy applicant for the December to March cohort. I noticed there was an error in the second block of code under the "Pyenv and setting up virtual-env" part of the guide. There was a letter 's' that was causing problems in the 'checking required packages' command.
I have corrected the error. This way other people who want to contribute to Apache won't face this issue I faced.
Thank you. :)

OyinOlamide committed:
Update CONTRIBUTORS_QUICK_START.rst

I am Oyinkansola Awosan, an Outreachy applicant for the December to March round.
This commit is to change the second block of code for 'Checking the required packages' under the 'Pyenv and setting up virtual-env' section.
The present block of code returns various errors when run, but the new proposed block of code runs smoothly without errors and would help contributors set up easily when they want to contribute to Apache Airflow.
Thank you.

uranusjr commented on CONTRIBUTORS_QUICK_START.rst:
```diff
@@ -136,8 +136,8 @@ Pyenv and setting up virtual-env
-  $ sudo apt install build-essentials python3.6-dev python3.7-dev python3.8-dev python3.9-dev python-dev openssl \
-       sqlite sqlite-dev default-libmysqlclient-dev libmysqld-dev postgresql
+  $ sudo apt install build-essential python3-dev openssl \
+       sqlite libsqlite3-dev default-libmysqlclient-dev libmysqlclient-dev postgresql
```
Actually, since we'll be installing Python with pyenv anyway, I don't think we even need the `python3-dev` package at all. Both `build-essential` and `libsqlite3-dev` can be removed as well since they are already installed in the command above.

potiuk replied:
Yeah I wonder where it came from :)

ephraimbuddy reviewed (changes requested):
Other places you should update too are:

- https://github.com/apache/airflow/blob/1571f80546853688778c2a3ec5194e5c8be0edbd/breeze#L276-L277
- https://github.com/apache/airflow/blob/c747bce135c5bcbc92645c226cb90fd99dc62be5/INSTALL#L10-L11

OyinOlamide committed:
Update CONTRIBUTORS_QUICK_START.rst

I am Oyinkansola Awosan, an Outreachy applicant for the December - March round.
This commit takes out some unnecessary parts in the second block of code under the 'Pyenv and setting up virtual-env' section.
This update will make setting up easier and smoother for contributors.
Thank you!

OyinOlamide committed:
Merge branch 'apache:main' into patch-1

OyinOlamide committed:
First commit

ephraimbuddy commented on INSTALL:
```diff
@@ -7,8 +7,8 @@ systems/prerequisites are known to work:
-sudo apt install build-essentials python3.6-dev python3.7-dev python-dev openssl \
-                 sqlite sqlite-dev default-libmysqlclient-dev libmysqld-dev postgresq
+sudo apt install openssl \
+                 sqlite default-libmysqlclient-dev libmysqld-dev postgresql
```
I _think_ we need `build-essential` and `libsqlite3-dev` here.

Apart from the contributors quick start doc. We need this here and also in breeze.
cc: @uranusjr

uranusjr replied:
Yes we do. The quick start does not need them because they are installed by a previous command, but that's not the case here.

ephraimbuddy replied:
@OyinOlamide

ephraimbuddy replied:
Also add `python3-dev` as well

OyinAwosan replied:
@ephraimbuddy I did not remove `libsqlite3-dev`
Do you want me to add it somewhere in the code?

ephraimbuddy replied:
Yes. add `build-essential python3-dev libsqlite3-dev` to the code you changed in airflow/INSTALL  and airflow/breeze

uranusjr replied:
You should add it here and the line you changed in `BREEZE`.

ephraimbuddy replied:
Just noticed you still have `libmysqlclient-dev` instead of libmysqlclient-dev

OyinAwosan replied:
@ephraimbuddy Sorry, but this change is not clear. Both strips of code you mentioned in your last comment seem to be the same or am I missing something?

ephraimbuddy replied:
Sorry, I meant `libmysqld-dev` instead of `libmysqlclient-dev`.

The change should be:
```
sudo apt install build-essential python3-dev libsqlite3-dev openssl \
                 sqlite default-libmysqlclient-dev libmysqlclient-dev postgresql
```

OyinAwosan replied:
@ephraimbuddy Alright, I was not aware of this before. Should this particular change take effect in all three files or just .breeze and INSTALL?

ephraimbuddy replied:
Just breeze and install

OyinAwosan replied:
I've done that now @ephraimbuddy 
Please check it out and let me know. Thank you for your patience and help.

OyinOlamide committed:
Second

OyinOlamide committed:
Third
